src/model/transformations/component.js

```diff
@@ -100,6 +142,20 @@ function remoteToState(remote, componentGroup, codesListsStore) {
     declarationMode,
   } = remote;
 
+  let redirectionclar = [];
+  if ( redirections != undefined ){
+    redirections.forEach(redirec => {
```

Comment:
why not using a filter ? 

redirectionClar = redirections.filter(r => r === r.flowControlType === undefined);